platformio_esp82xx_envs.ini

```diff
@@ -452,6 +458,8 @@ platform_packages         = ${ir.platform_packages}
 lib_ignore                = ${ir.lib_ignore}  
 build_flags               = ${minimal_ir_extended.build_flags}
                             ${esp8266_4M2M.build_flags}
+extra_scripts             = ${esp8266_4M2M.extra_scripts}
```

Comment:
`esp8266_4M2M` doesn't have `extra_scripts`
The entire sequence of envs must have all items you want to change/extend.
Just having `extends` isn't enough.... sadly.